### DISCUSSION TITLE
Freetz für 7490 und FRITZ!OS 7.51-102214 (Beta)

### DISCUSSION THREAD

**Discussion by eDonkeyElseWhere** [Freetz-NG/freetz-ng#694] [Allgemein] [answered]
Warum wird bei der 7490 eigentlich nicht auch die öffentliche Beta angeboten?

**Comment by fda77** [comment]
Und wo kommt das Stroh her?

**Reply by eDonkeyElseWhere** [reply]
Falls Du damit meinst, das per JuisCheck in der Richtung nichts gefunden wird, dann akzeptiere ich das so. Ansonsten gibt es die Beta ja offiziell hier.
[Frisch aus der Entwicklung](https://avm.de/fritz-labor/frisch-aus-der-entwicklung/frisch-aus-der-entwicklung/)

**Reply by fda77** [reply]
Ne, das war eher frei nach ["Warum liegt hier Stroh?"](https://youtu.be/6foTWmryAvQ)

**Comment by fda77** [comment, marked as answer]
https://github.com/Freetz-NG/freetz-ng/commit/ed8dca5159c7916461b899eefe93c7db2f1c6801

**Reply by eDonkeyElseWhere** [reply]
Danke, hat funktioniert!

**Reply by fda77** [reply]
Es ist wirklich komisch dass juis da nichts findet, es scheint die erste und einzige Labor der 7490 zu sei, seit fast 1 Monat.
Sind da neue dsl treiber drin? Ich hab so ne grottige Telekom Leitung

**Reply by eDonkeyElseWhere** [reply]
Werde berichten, wenn sie läuft.

**Reply by eDonkeyElseWhere** [reply]
Firmware ist drauf, wie stabil sie ist muss man abwarten.

**Reply by fda77** [reply]
Jo, mach mal. Ich kann nicht viel mit der 7490 testen. Ich hab von der Terrorkom schon ne Dauerdrossel. Und wenn experimentiere wird noch mehr gedrosselt

**Comment by eDonkeyElseWhere** [comment]
Läuft jetzt seit 6 Tagen stabil, MagentaZuhause S 16/2,4 mit  22,6/3,3 an  VDSL2 17a G.Vector. Download auf Anschlag.
